### PR TITLE
Feature/sas 4866/circleci integration

### PR DESCRIPTION
**Pull request by gguridi**
This change is [Reviewable](https://reviewable.io/reviews/affectv/ingesters-kafka/1)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+test-step: &test-step
+  machine: true
+  steps:
+    - checkout
+    - run:
+        name: Run integration tests
+        command: |
+          echo "true"
```
Comment (disi1166) [inline .circleci/config.yml]: I assume we don't have integration tests yet, but there are some unit-tests, shouldn't we have a step to run those?

**Comment by garciapuig** [comment]
What's this reviewable thing? :eyes:

**Comment by gguridi** [comment]
@disi1166 ready to review :)

File: sharethis/kafka_push.py [hunk 2]
```diff
@@ -70,9 +77,11 @@ def transform(self, input_json):
             'al': None,
             'ua': input_json.get('userAgent'),
             'opt': {
-                'ref': input_json.get('refDomain')
-            }
+                'ref': input_json.get('refDomain'),
+            },
         }
+        transformed['opt'].update(self.get_unrecognised_parameters(input_json))
```
[the remark this answers is not in the document]
Comment (gguridi) [reply sharethis/kafka_push.py]: Yes, that's part of SAS-4703 changes ported to the new repository.

**Pull request merged**